ui/apps/platform/cypress/integration/clusters/Clusters.helpers.js

```diff
@@ -43,7 +43,7 @@ const routeMatcherMapForDelegateScanning = {
 };
 
 const basePath = '/main/clusters';
-const delegateScanningPath = `${basePath}/delegate-scanning`;
+export const delegatedScanningPath = `${basePath}/delegated-image-scanning`;
```

Comment:
Although verbose, the more similar the better to UI title and heading, also `describe` block title below.
* `delegatedImageScanningPath`
* `visitDelegatedImageScanning`

Not a blocker to merge.

Reply:
I actually considered changing the var name to match, but decided against it because always trying to keep the var name equal to its assigned value--while easier with modern editor tooling--seems to be unnecessary.